app/pages/settings/email.cjsx

```diff
@@ -12,6 +13,28 @@ module.exports = React.createClass
   render: ->
     <div className="content-container">
       <p><strong>Project email preferences</strong></p>
```

Comment:
I think this title should be able the lower group of email preferences. And this line should be "Zooniverse email preferences".